be/src/service/service_be/starrocks_be.cpp

```diff
@@ -0,0 +1,94 @@
+// This file is made available under Elastic License 2.0.
```

Comment:
This file is newly created, and most content is rewritten. I think it is OK to not keep origin license header.